model/ExpandedLicensing/Classes/IndividualLicense.md

```diff
@@ -1,21 +0,0 @@
-SPDX-License-Identifier: Community-Spec-1.0
-
-# IndividualLicense
```

Comment:
IndividualLicenseInfo instead of IndividualLicense

Restore this class

Reply:
IndividualLicensingInfo... since it's not info on a license